README.md

```diff
@@ -129,15 +129,15 @@ names are ignored.
 ### `Pkgfile`
 
 ```yaml
-# syntax = docker.io/autonomy/bldr:1289eba-frontend
+# syntax = docker.io/autonomy/bldr:v0.1.0-alpha.0
```

Comment:
just to verify this is intentional?

Reply:
yeah, it was, but I guess I used wrong tag without `-frontend`

README was written before we had a first formal release